angr/storage/memory_mixins/paged_memory/pages/ultra_page.py

```diff
@@ -98,12 +94,13 @@ def fill(end):
             result = self._force_load_cooperation(result, size, endness, memory=memory, **kwargs)
         return result
 
-    def store(self, addr, data, size=None, endness=None, memory=None, page_addr=None, cooperate=False, **kwargs):
+    def store(self, addr, data: Union[int,SimMemoryObject], size=None, endness=None, memory=None, page_addr=None,
+              cooperate=False, **kwargs):
         if not cooperate:
             data = self._force_store_cooperation(addr, data, size, endness, memory=memory, **kwargs)
 
-        if data.object.op == 'BVV':
-            # mark range as symbolic
+        if type(data) is int or data.object.op == 'BVV':
```

Comment:
when will this branch ever be taken? force_store_cooperation should be making sure that data is forced to be a SimMemoryObject

Reply:
I explicitly call this method to store a concrete integer into memory with `cooperate=True` to avoid the unnecessary conversion from int to a BVV.